packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -136,13 +136,10 @@ local function request(url, accept_404_reply, auth_token)
     return res, nil
 end
 
-local function is_ip_per_task(app)
-    return app["ipAddress"] ~= nil and app["ipAddress"] ~= cjson_safe.null
-end
-
-local function is_user_network(app)
+local function is_container_network(app)
     local container = app["container"]
-    return container and container["type"] == "DOCKER" and container["docker"]["network"] == "USER"
+    local network = app["networks"][1]
+    return container and network and network["mode"] == "container"
```

Comment:
probably want something like this:
```
(network["mode"] == "container" or network["mode"] == "container/bridge")
```

also wondering why test cases didn't pick this up since "container/bridge" is the network type used in the mock... bad/incomplete tests? bridge-mode is a subset of container-mode (when in bridge mode, apps are required to specify a host-port for each port-mapping; host-port is optional when in non-bridged container mode)

Reply:
> probably want something like this:
> 
> ```
> (network["mode"] == "container" or network["mode"] == "container/bridge")
> ```
> also wondering why test cases didn't pick this up since "container/bridge" is the network type used in the mock... bad/incomplete tests? bridge-mode is a subset of container-mode (when in bridge mode, apps are required to specify a host-port for each port-mapping; host-port is optional when in non-bridged container mode)

Interestingly, the current code works with both container-mode and bridge mode: https://jira.mesosphere.com/browse/DCOS-45468?focusedCommentId=198613&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-198613 (tested manually)

the unit test worked because `test_app_with_container_networking_and_portmappings()` defines:
```app['networks'] = [{
            'mode': 'container',
            'name': 'samplenet'
        }]
```
thus, overriding the container-bridge mode.

I will add another unit test for container-bridge mode.